xrviz/tests/test_dashboard.py

```diff
@@ -211,3 +211,11 @@ def test_create_taps_and_series_graph_for_2d_coords(dashboard):
     dashboard.create_graph()
     dashboard.create_taps_graph(x=-79.232, y=43.273)
     assert isinstance(dashboard.series_graph[0], pn.pane.holoviews.HoloViews)
+
+
+@pytest.mark.skipif(not has_crick_tdigest, reason='crick.tdigest not present')
+def test_find_cmap_limits_with_crick_tdigest():
+    ds = xr.tutorial.open_dataset('air_temperature',
+                                  chunks={'lat': 25, 'lon': 25, 'time': 10})
+    a, b = find_cmap_limits(ds.air)
+    assert (a, b) == (255.38780056044027, 298.5900340551101)
```

Comment:
These numbers should not the exact like this, precision may differ by system. Probably it is good enough if `abs(expected - found) < 0.1`

Reply:
Corrected. Thanks!